commands/mentor.py

```diff
@@ -0,0 +1,28 @@
+import requests
+import random
+from disco.bot import Bot,Plugin
+from disco.types.message import MessageEmbed
+from config_mock import AnnounceBotConfig, TestPluginConfig
+
+MENTOR_ID = 487807730506006540
+
+@Plugin.with_config(TestPluginConfig)
+class MentorPlugin(Plugin):
+
+    def get_avail_mentors(self):
+       return [u.user.id for u in self.bot.client.state.guilds.get(487611837664198679).members.values() if MENTOR_ID in u.roles and u.user.presence.status != "OFFLINE"]
```

Comment:
This line seems like it would take some time to execute, considering how many people are currently in Discord Testers. Maybe caching it would be helpful?